src/lib/efreet/efreet_trash.c

```diff
@@ -30,11 +26,6 @@ static int _efreet_trash_log_dom = -1;
 static unsigned int _efreet_trash_init_count = 0;
 static const char *efreet_trash_dir = NULL;
 
-#ifdef _WIN32
-# define getuid() GetCurrentProcessId()
-#endif
-
-
```

Comment:
Why removing this?

Reply:
[line 13](https://github.com/expertisesolutions/efl/pull/197/files/9c6caa5efd1855d134b2e3857f755f0a6abcf210#diff-f9abea168c370e10cbeaabb205fb0ccbR13) already defines it.

Reply:
Why not removing the one from line 13? I think it make more sense as in <headers> <functions> separation.